AdyenCard/Components/Card/CardDetails.swift

```diff
@@ -92,7 +92,6 @@ public struct CardDetails: PaymentMethodDetails, BillingAddressInformation {
         case encryptedSecurityCode
         case holderName
         case fundingSource
-        case billingAddress
```

Comment:
was this a bug in the released version?

Reply:
`billingAddress` is duplicated in payment data, but hopefully it is not causing any troubles (at least on TEST)

Reply:
Like 

```json

{
  ...
  "paymentMethod" : {
    ...
    "billingAddress" : {
      "postalCode" : "123456",
      "country" : "ZZ",
      "street" : "null",
      "houseNumberOrName" : "null",
      "city" : "null",
      "stateOrProvince" : "null"
    },
  },
  "billingAddress" : {
    "postalCode" : "123456",
    "country" : "ZZ",
    "street" : "null",
    "houseNumberOrName" : "null",
    "city" : "null",
    "stateOrProvince" : "null"
  },
  ...
}
```